cypress/e2e/tests/pages/data/agent-configuration-payload.ts

```diff
@@ -212,7 +212,6 @@ export const payloadComparisonData = {
           {
             podAffinityTerm: {
               namespaces:    null,
-              weight:        10,
```

Comment:
When the rke2 agent config tests were written we we sending incorrect weighted affinity terms -[ I fixed node affinity and pod antiAffinity,](https://github.com/rancher/dashboard/pull/9011) missed podAffinity and noticed that running these tests.